scripts/rmc

```diff
@@ -41,6 +46,10 @@ def main():
 
     if not rmc.dependencies_in_path():
         return 1
+    
+    # Add some CBMC flags by default unless `--no-default-checks` is being used
+    if not args.no_default_checks:
+        rmc.add_default_cbmc_flags(args)
```

Comment:
it wasn't clear to me from the name that this only adds some default flags (controlled by args)

Reply:
Not sure I follow. What changes to this are you suggesting?

Reply:
naming it something like `add_selected_default_cbmc_flags`

Reply:
Done.